pages/hardware/devices.md

```diff
@@ -6,128 +6,88 @@ title: Supported Devices
-[link](http://www.solid-run.com/products/hummingboard/)
-
-![Humming Board](/img/hummingBoard.png)
-
-### Nitrogen6x by Boundary Devices
-
-[link](http://boundarydevices.com/product/nitrogen6x-board-imx6-arm-cortex-a9-sbc/)
-
-![Nitrogen6x](/img/nitrogen6x.jpg)
-
-### NXP/Freescale i.MX6 Sabre Lite
-
-[link](http://boundarydevices.com/product/sabre-lite-imx6-sbc/)
-
-![i.MX6 Sabre](/img/iMX6-sabre-lite.jpg)
-
-### Odroid C1+
-
-[link](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143703355573)
-
-![Odroid C1+](/img/odroidC1Board.jpg)
-
-### ODROID XU4
-
-[link](http://www.hardkernel.com/main/products/prdt_info.php?g_code=G143452239825&tab_idx=1)
-
-![ODROID XU4](/img/odroidXUBoard.jpg)
-
-### Zynq ZC702 from Xilinx
-
-[link](http://www.xilinx.com/products/boards-and-kits/ek-z7-zc702-g.html)
-
-![Zynq board](/img/zynq.jpg)
-
-### VIA VAB-820
-
-[link](http://www.viatech.com/en/boards/pico-itx/vab-820/)
-
-![VIA VAB-820](/img/viaVab820.jpg)
-
-### Intel NUC
-
-[link](http://www.intel.com/content/www/us/en/nuc/overview.html)
-
-![NUC](/img/nuc.jpg)
-
-## Beta Support
-
-### BeagleBone Green Wireless
-
-[link](http://www.seeedstudio.com/wiki/SeeedStudio_BeagleBone_Green_Wireless)
-
-![BeagleBone Gren Wireless](/img/beaglebone-green-wireless.jpg)
-
-### Samsung Artik 5
-
-[link](https://www.artik.io/modules/overview/artik-5/)
-
-![Samsung Artik 5](/img/artik5/samsung-artik-5.jpg)
-
-### Samsung Artik 10
-
-[link](https://www.artik.io/modules/overview/artik-10/)
-
-![Samsung Artik 10](/img/artik10/samsung-artik-10.jpg)
-
-### Toradex Colibri iMX6DL
-
-[link](http://developer.toradex.com/products/colibri-imx6)
-
-![Toradex Colibry iMX6DL](/img/toradex-colibri-imx6dl.jpg)
-
-### Technologic TS-4900
-
-[link](https://www.embeddedarm.com/products/TS-4900)
-
-![Technologic TS-4900](/img/ts-4900.jpg)
+Currently we support over 22 single board computers (as well as some of their variants) and are capable of supporting customer boards for customers if requested. The boards are split into 3 different support tiers. The 1st level are the boards with the best support and we try to guarantee that all the hardware peripherals and drivers are working. These boards should work flawlessly like the would with Debian OS or the operating system the board ships with. For the 2nd tier boards, most of the hardware peripherals should work, but we can't guarantee complete compatibility and you may find a few things that don't work.
+
+For the 3rd tier boards, we only guarantee that the machines boot and basic functionality is available. If you find any issues with hardware compatibility with tier 2 or 3 please report the issue on github.com/resin-os/resinos and we will try provide support.
+
+If you think there is a popular or interesting widely available board that you think we should add support for, please create a GitHub issue in the
+[resin-os-device-support](https://github.com/resin-os/resin-os-device-support) repository and we will see what we can do.
```

Comment:
Should we encourage people to vote on the boards as well? (so we can judge popularity better)